src/process_utility.c

```diff
@@ -752,6 +752,24 @@ rename_hypertable_constraint(Hypertable *ht, Oid chunk_relid, void *arg)
 	chunk_constraint_rename_hypertable_constraint(chunk->fd.id, stmt->subname, stmt->newname);
 }
 
+static void
+alter_hypertable_constraint(Hypertable *ht, Oid chunk_relid, void *arg)
+{
+	AlterTableCmd *cmd = (AlterTableCmd *) arg;
+	Constraint *cmd_con;
+
+	cmd_con = castNode(Constraint, cmd->def);
```

Comment:
Have we officially dropped support for PG < 9.6.2?  If I recall, `castNode` is not available pre 9.6.2. Also seems inconsistent to use it here but not for `AlterTableCmd` above (nor in many other places).

Reply:
Also, can move up to previous line.